### PR TITLE
CIV-6214 dahsboard status part2

### PR DESCRIPTION
**Pull request by TatsyMaslova**
### JIRA link ###

https://tools.hmcts.net/jira/browse/CIV-6214

### Change description ###

This pr is against the branch that it is dependant on and is currently in test.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ x] No
```

### CODE REVIEW

File: src/test/unit/routes/features/dashboard/dashboardController.test.ts [hunk 1]
```diff
@@ -1,67 +1,34 @@
...
 
   describe('on GET', () => {
     it('should return dashboard page', async () => {
-      await testSession
-        .get(DASHBOARD_URL)
-        .expect((res: Response) => {
-          expect(res.status).toBe(200);
-          expect(res.text).toContain('Claims made against you');
-        });
+      // await  request(app)
```
Comment (TatsyMaslova) [inline src/test/unit/routes/features/dashboard/dashboardController.test.ts:26]: this is temporarily will revert all changes here

File: src/main/routes/features/dashboard/dashboardController.ts [hunk 2]
```diff
@@ -26,12 +26,11 @@ function renderPage(res: Response, claimsAsClaimant: DashboardClaimantItem[], cl
...
-dashboardController.get(DASHBOARD_URL, async function (req: AppRequest, res) {
-  const user: UserDetails = req.session.user;
-  /*This is a call to validate integration with legacy draft-store. This will have to be refined in the future
-  to display the draft claims on the dashboard*/
-  const claimsAsClaimant : DashboardClaimantItem[] = await civilServiceClient.getClaimsForClaimant(req);
-  const claimsAsDefendant : DashboardDefendantItem[] = await civilServiceClient.getClaimsForDefendant(<AppRequest>req);
+dashboardController.get(DASHBOARD_URL, async function (req, res) {
+  const appRequest = <AppRequest> req;
+  const user: UserDetails = appRequest.session.user;
+  const claimsAsClaimant : DashboardClaimantItem[] = await getClaimsForClaimant(appRequest);
```
Comment (laarmada) [inline src/main/routes/features/dashboard/dashboardController.ts:32]: only a general comment in some point we need to catch that call's with try/catch no?

**Review by laarmada — APPROVED** [review]

File: src/main/common/models/dashboard/dashboardItem.ts [hunk 3]
```diff
@@ -1,7 +1,25 @@
 import config from 'config';
+import {getLng} from 'common/utils/languageToggleUtils';
+import {t} from 'i18next';
+import {formatDateToFullDate} from 'common/utils/dateUtils';
+import {getLanguage} from 'modules/i18n/languageService';
 
 const ocmcBaseUrl = config.get<string>('services.cmc.url');
 
+export interface DashboardStatusTranslationParam {
```
Comment (PaulH-HMCTS) [inline src/main/common/models/dashboard/dashboardItem.ts:9]: Maybe we should just have a generic KeyValue interface here that we can use elsewhere

File: src/main/services/features/dashboard/dashboardService.ts [hunk 4]
```diff
@@ -0,0 +1,14 @@
+import config from 'config';
+import {CivilServiceClient} from 'client/civilServiceClient';
+import {AppRequest} from 'models/AppRequest';
+
+const civilServiceApiBaseUrl = config.get<string>('services.civilService.url');
+const civilServiceClient: CivilServiceClient = new CivilServiceClient(civilServiceApiBaseUrl);
+
+export const getClaimsForClaimant = async (appRequest: AppRequest) => {
+  return   await civilServiceClient.getClaimsForClaimant(appRequest);
```
Comment (PaulH-HMCTS) [inline src/main/services/features/dashboard/dashboardService.ts:9]:
```suggestion
  return await civilServiceClient.getClaimsForClaimant(appRequest);
```

File: src/test/unit/app/client/legacyDraftStoreClient.test.ts [hunk 5]
```diff
@@ -4,6 +4,10 @@ import axios from 'axios';
 
 jest.mock('../../../../main/app/client/serviceAuthProviderClient');
 jest.mock('axios');
+jest.mock('../../../../main/modules/i18n/languageService', ()=> ({
```
Comment (PaulH-HMCTS) [inline src/test/unit/app/client/legacyDraftStoreClient.test.ts:7]:
```suggestion
jest.mock('../../../../main/modules/i18n/languageService', () => ({
```

**Pull request closed without merging**